### PR TITLE
Update Snaps tutorials to match latest tooling

### PR DESCRIPTION
**Pull request by Montoya**
The Snaps tutorials were out of date with the latest dependencies, tooling, and visual style of Snaps. This updates everything to latest requirements.

### CODE REVIEW

**Comment by joaniefromtheblock** [comment]
Hi @Montoya. Thank you for opening. Running through this tutorial I am getting an error the following error at step 4. 

```
An error happened: Request of type 'wallet_requestPermissions' already pending for origin http://localhost:8000. Please wait.
```

**Comment by Montoya** [comment]
> Hi @Montoya. Running through these steps, I get the following error when I try to send a message:
> 
> ```
> An error happened: Unauthorized to perform action. Try requesting the required permission(s) first. For more information, see: https://docs.metamask.io/guide/rpc-api.html#permissions
> ```

On which tutorial? Are you sure the Snap installed correctly?

**Review by joaniefromtheblock — APPROVED** [review]
LGTM

File: snaps/learn/tutorials/gas-estimation.md [hunk 1]
```diff
@@ -199,10 +257,9 @@ Similarly, you should update the `name`, `version`, `description`, and `reposito
...
 
-Lastly, you can update the content of `/packages/site/src/pages/index.tsx`, such as changing the
+You can update the content of `/packages/site/src/pages/index.tsx` by changing the
 name of the method for showing gas fee estimates.
-If you change the method name, make sure to change the method name in `/packages/snap/src/index.ts`
-to match.
+If you change the method name in `/packages/site/src/pages/index.tsx`, ensure you change the method name in `/packages/snap/src/index.ts` to match.
 
 After you have made all necessary changes, you can
 [publish your Snap to npm](../../how-to/publish-a-snap.md#publish-your-snap).
```
Comment (alexandratran) [inline snaps/learn/tutorials/gas-estimation.md:265]: Update link to `../../how-to/publish-a-snap.md`.

**Review by alexandratran — COMMENTED** [review]
@joaniefromtheblock Minor comments to retroactively address.

File: snaps/learn/tutorials/gas-estimation.md (hunk 1 above)
Comment (joaniefromtheblock) [reply snaps/learn/tutorials/gas-estimation.md:265]: Thanks @alexandratran will address in new PR

File: snaps/learn/tutorials/transaction-insights.md [hunk 2]
```diff
@@ -127,6 +146,12 @@ export const onTransaction: OnTransactionHandler = async ({ transaction }) => {
 };
 ```
 
+:::tip
+If you have previously developed a decentralized app (dApp), you're likely familiar with 
+accessing the Ethereum provider using `window.ethereum`. In a Snap, the `window` object is not available.
+Instead when you request the `endowment:ethereum-provider` permission, your Snap is granted access to the `ethereum` global object.
```
[the remark this answers is not in the document]
Comment (joaniefromtheblock) [reply snaps/learn/tutorials/transaction-insights.md:152]: Thanks @alexandratran will address in new PR

**Pull request merged**